lib/Serialization/Deserialization.cpp

```diff
@@ -143,14 +143,9 @@ static void skipRecord(llvm::BitstreamCursor &cursor, unsigned recordKind) {
   auto next = cursor.advance(AF_DontPopBlockAtEnd);
   assert(next.Kind == llvm::BitstreamEntry::Record);
 
-#if NDEBUG
-  cursor.skipRecord(next.ID);
-#else
-  SmallVector<uint64_t, 64> scratch;
-  StringRef blobData;
-  unsigned kind = cursor.readRecord(next.ID, scratch, &blobData);
+  unsigned kind = cursor.skipRecord(next.ID);
```

Comment:
I assume there was an upstream API change to return the `kind` from `skipRecord()` that we hadn't adopted until now?

Reply:
Presumably! I didn't bother to spelunk through LLVM's history in this case, only saw that we can do better.